### ISSUE TITLE
Padding incorrect in Promo Card

### ISSUE THREAD

**Issue by justintemps** [international-labour-organization/designsystem#569]
Check designs.

![highlighted_padding_stager](https://github.com/international-labour-organization/designsystem/assets/12401179/ea31dfd8-c2ea-4f0c-8c35-058fe04560a1)

**Comment by bashlk** [comment]
@justintemps  I checked the paddings within the promo card against the design (https://www.figma.com/file/EIlsB8YqRXKwk7SXj5sOVp/ILO_Components?type=design&node-id=7191-14643&mode=design&t=rPlAc0wKCATdMmxp-4) and they all seem to match.

<img width="1284" alt="Screenshot 2023-11-08 at 13 10 15" src="https://github.com/international-labour-organization/designsystem/assets/11556301/094b92aa-a9dc-4477-817c-89fc55ea9b67">

The strange thing I found is that the element is getting a padding by default which is slightly different from the "standard" version which is marked in wingsuit as the default. I assume that setting the default value in wingsuit with the preview property does not ensure that it is set by default when the component is used outside of storybook.

<img width="587" alt="Screenshot 2023-11-08 at 13 06 25" src="https://github.com/international-labour-organization/designsystem/assets/11556301/e2ff43f0-5729-4544-a28e-0e0070bdfa71">
